composer.json

```diff
@@ -18,14 +18,14 @@
         "issues": "https://github.com/oscarotero/form-manager/issues"
     },
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": ">=7.2",
```

Comment:
I guess 7.1 compatibility is asking too much, right?

Reply:
Yes, specially if we want to upgrade to v5 of the validator, it's the only possibility to support 7.2. v4 has many red alerts.

Reply:
fair enough!
This will require a major version then.